assets/css/_docs.scss

```diff
@@ -160,7 +160,7 @@ html.docs {
     input {
       border: none;
       border-bottom: 2px solid $defaultLiteColor;
-      font-size: 1.3rem;
+      font-size: 16px;
```

Comment:
I can't repro. When I set the font-zie of the input to 1.6rem I get this computed result:
<img width="653" src="https://cloud.githubusercontent.com/assets/4303/18595916/a0ea8102-7bfb-11e6-99ba-852bad47896d.png">